src/core/test/system/borrower/balance.t.sol

```diff
@@ -23,11 +23,14 @@ contract BalanceTest is SystemTest {
     Borrower borrower;
     address borrower_;
         
+    SwitchableDistributor distributor;
     function setUp() public {
         baseSetup();
         // setup users
+        distributor = SwitchableDistributor(address(lenderDeployer.distributor()));
```

Comment:
SwitchableDistributor(address(distributor));

Reply:
I think thats not possible